docs/DeveloperGuide.adoc

```diff
@@ -46,6 +46,10 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |`* * *` |user |add a new person |
 |`* * *` |user |delete a person |remove entries that I no longer need
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |user |update a person by name |make necessary changes to a person's details in the list
+|`* * *` |user |insert an alternative phone number for a person |contact a person in other ways
+|`* *` |user |add remarks to a person's contact |remember how I know this person (not contacted often)
```

Comment:
You can try to think of user stories in which the role is more specific than `user`.